@shared/api/helpers/extensionMessaging.ts

```diff
@@ -1,11 +1,10 @@
-import { browser } from "webextension-polyfill-ts";
+import browser from "webextension-polyfill";
```

Comment:
`webextension-polyfill-ts` is deprecated. The latest has been rolled into `webextension-polyfill`. I was having trouble building without upgrading to the latest